index.js

```diff
@@ -46,6 +47,9 @@ class MMST extends EventEmitter {
     // How long to lookup peers fore before giving up and using what you have
     lookupTimeout = DEFAULT_LOOKUP_TIMEOUT,
 
+    // How long to wait for `run` to be finished
+    queueTimeout = lookupTimeout + (2 * 1000),
```

Comment:
Mind putting this default into a const at the top instead?

Reply:
:+1:

Reply:
Oh wait, having it dynamic might be better. Does it actually pull the value from lookupTime? I'm not sure how the spread operator works with defaults. 😅

Reply:
me neither haha I just try it and it worked